ColdWeatherBoyy/dead-inside-games
# 31 html route for login page login

❗ Please **DO NOT** open a Pull Request without creating an Issue first. Failure to do so will result in the rejection of the pull request.

## Purposegit 

Adds HTMLRoutes for login page and sign up page, as well as updates HelloWorld route to 404 page.

## Changes

Changes are to htmlRoutes.js file mostly. Added a handlebars called errormessage.handlebars and an image in public/images.

## Steps to Reproduce or Test

Visit any non-designated URL path/you can also see the handelbars error when going to /login or /signup as those handlebars templates don't exist yet.

## Optional GIF of screenshot


![Screenshot 2023-03-22 at 1 26 59 PM](https://user-images.githubusercontent.com/123032126/226988159-7150b46a-ef61-47ca-b776-f5f0bfb3f6a3.png)
